### PR TITLE
[mri violations] MRI Violations module does not describe or test widgets

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

- [x] Have you updated related documentation?

#### Testing instructions (if applicable)

1. Follow the README.md and TestPlan.md instructions for the mri_violations module.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
#6440

### CODE REVIEW

File: modules/mri_violations/README.md [hunk 1]
```diff
@@ -85,3 +85,8 @@ the problem "Could not identify scan type"
 "Protocol Violation"
 - `MRICandidateErrors` logs all the scans referring to the problem
 "Candidate Mismatch"
+
+## Dashboard Widget
+
+The total `Violated scans` will be shown inside the Dashboard widget, named "My Tasks"
+and when clicked will link to the mri_violations module.
```
Comment (PapillonMcGill) [inline modules/mri_violations/README.md:92]:
```suggestion
with a link pointing to the mri_violations module.
```

File: modules/mri_violations/test/TestPlan.md [hunk 2]
```diff
@@ -50,6 +50,12 @@
    that the help content about MRI Protocol Check Violation is showing up
    and is up-to-date.
 
+### Dashboard Widget - "My Tasks" for Violated scans
+1. Ensure the total of Violated scans corresponds with the correct
```
Comment (driusan) [inline modules/mri_violations/test/TestPlan.md:54]: Are there site or permission related things that need to be verified while testing?

Comment (maltheism) [reply modules/mri_violations/test/TestPlan.md:54]: Hi @driusan, I added a line about what permissions are needed from the mri_violations module to view the widget.

Comment (driusan) [reply modules/mri_violations/test/TestPlan.md:54]: Does the text of the label change depending on the permission? Or the filtering automatically done when clicking on the link?

Comment (maltheism) [reply modules/mri_violations/test/TestPlan.md:54]: @driusan the Violated scans row in My Tasks will not display if the user doesn't have either of the permissions.

Comment (driusan) [reply modules/mri_violations/test/TestPlan.md:54]: Yeah, but in most of them the label changes from "Sites: all" to "Sites: user sites".. is that not the case for this one?

Comment (nicolasbrossard) [reply modules/mri_violations/test/TestPlan.md:54]: @driusan The label is always 'Violated scan(s)' for this widget.

Comment (maltheism) [reply modules/mri_violations/test/TestPlan.md:54]: @driusan see @nicolasbrossard comment.

**Pull request merged**